front/src/styles/scss/applications/operationalStudies/_scenario.scss

```
@@ -32,21 +32,21 @@
 
     display: inline-flex;
     align-self: flex-start;
-    height: 2rem;
-    font-size: 0.875rem;
-    letter-spacing: 0.1019rem;
-    line-height: 1.125rem;
-    margin-block: 1.813rem 2.063rem;
+    height: 32px;
+    font-size: 14px;
+    letter-spacing: 1.6304px;
+    line-height: 18px;
+    margin-block: 29.008px 33.008px;
```

Can you round the values ? 
```suggestion
    margin-block: 29px 33px;
```

---

Right, fractional pixel values tend to cause issues and are not handled consistently by all browsers.